### DISCUSSION TITLE
Downloaded example scripts are missing files

### DISCUSSION THREAD

**Discussion by cjdupuis1** [JSBSim-Team/jsbsim#683] [Q&A]
I have been having trouble with a lot of the example scripts that come with the download. For example, the Short-S23 is missing the Short_S23-hydrodynamics system file. The ah1s helicopter model is missing files. It seems like only about half of the example scripts run without errors. I can provide more examples if necessary.

Are all of the example scripts supposed to run?

**Comment by seanmcleod** [comment]
Hmm, `Short_S23-hydrodynamics.xml` does exist in the repo - https://github.com/JSBSim-Team/jsbsim/blob/master/aircraft/Short_S23/Systems/Short_S23-hydrodynamics.xml

Note it's in the `Systems` sub-directory of the aircraft as opposed to the global `Systems` directory.

Which ah1s files are missing?

**Reply by cjdupuis1** [reply]
Thanks so much for replying. I will also look at that link, perhaps my download was flawed and I am missing files.

**Reply by cjdupuis1** [reply]
It does look like my download is missing all of the subdirectory system folders. I used "pip install jsbsim", maybe that doesn't include all the systems files? I think I will just clone the current repository and update my files. Thanks again!

**Reply by seanmcleod** [reply]
@cjdupuis1 yes I'd suggest creating a new discussion with a subject line that makes sense for the new set of questions etc. you have. Makes it easier for others in future to find/come across if they have similar questions etc.

**Comment by cjdupuis1** [comment]
Cloning the GitHub repository and replacing the pip install jsbsim aircraft, engine, scripts, and systems folders

**Comment by seanmcleod** [comment]
I just double-checked the contents of the wheel file and confirmed that no sub-directories like `Systems` under an aircraft directory are bundled in the wheel file. So this is something we should fix. I double-checked `ah1s` and `Short_S23`.

**Reply by cjdupuis1** [reply]
Great, glad it wasn't just me making a silly mistake.

**Reply by bcoconni** [reply]
Good catch @seanmcleod, the code that builds the Python wheels does not include the `Systems` sub-directory located under aircraft when there is one:
https://github.com/JSBSim-Team/jsbsim/blob/82ac929c439b8b23c774c786bf9c7ff6efee7561/python/setup.py.in#L218-L223

Thanks for reporting the issue @cjdupuis1 👍

**Reply by bcoconni** [reply]
@cjdupuis1 FYI I have created the issue #687 from your original comment.
Thanks for the bug report.

**Comment by seanmcleod** [comment]
@bcoconni in addition to aircraft specific `Systems` sub-directories some aircraft also include an `Engines` sub-directory, e.g. the `ah1s` that I checked when I looked into this issue.

**Reply by bcoconni** [reply]
Yes, indeed. I just pushed a commit that should fix that.
